CargoBay/CargoBay.h

```diff
@@ -169,6 +169,16 @@
  */
 - (void)setPaymentQueueUpdatedDownloadsBlock:(void (^)(SKPaymentQueue *queue, NSArray *downloads))block;
 
+
+/**
+ Sets a block to be called when the payment has requested directly via the App Store.
```

Comment:
the payment _was_ requested? Or _has been_ requested, maybe?

Reply:
I didn't write this code.

Reply:
done, thx

Reply:
It's also not a big deal, btw.